fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Literal.java

```diff
@@ -109,7 +109,7 @@ public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
 
     @Override
     public String sql() {
-        return null;
+        return value.toString();
```

Comment:
Should we break them down into separate expressions @EmmyMiao87 ?

Reply:
Added todo information, which is subsequently rewritten in subexpressions